client/src/components/DataFiles/DataFilesProjectFileListing/DataFilesProjectFileListing.module.scss

```diff
@@ -33,6 +33,13 @@
   align-items: flex-start;
 }
 
+.title {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  max-width: 1400px;
+}
```

Comment:
__Can Be More DRY__

See [previous comment about how we can add truncation styles](https://github.com/TACC/Core-Portal/pull/444/files#diff-7ff4ce9de2c3aacd50469d7f92f6a3e1b4a35de8f3e1595ccfde2dae2176b29dR4-R6).

---

Also, rather than add truncation rules as needed for any section table header, consider letting [all `<SectionTableWrapper>` headers a.k.a. titles][section-table-wrapper-header] be truncated (and maybe even [all `<Section>` headers a.k.a. titles][section-header]).
- This change will require testing all sections, which would increase the scope of this ticket.
- If the scope is too large for this ticket to accommodate, then you could create a new ticket to:
    - move `<SectionTableWrapper>` header truncation to [one place]()
    - move `<Section>` header truncation to [one place][section-header]

[section-table-wrapper-header]: https://github.com/TACC/Core-Portal/blob/main/client/src/components/_common/SectionTableWrapper/SectionTableWrapper.module.css#L11-L14
[section-header]: https://github.com/TACC/Core-Portal/blob/main/client/src/components/_common/Section/Section.module.css#L16-L21

Reply:
I have removed my comment about a larger-scope solution.